widget/form.go

```diff
@@ -371,3 +370,21 @@ func NewForm(items ...*FormItem) *Form {
 
 	return form
 }
+
+type formItemLayout struct{}
+
+func (f formItemLayout) Layout(objs []fyne.CanvasObject, size fyne.Size) {
+	itemHeight := objs[0].MinSize().Height
+	objs[0].Resize(fyne.NewSize(size.Width, itemHeight))
+
+	objs[1].Move(fyne.NewPos(theme.InnerPadding(), itemHeight+theme.InnerPadding()/2))
+	objs[1].Resize(fyne.NewSize(size.Width, objs[1].MinSize().Width))
+}
+
+func (f formItemLayout) MinSize(objs []fyne.CanvasObject) fyne.Size {
+	min0 := objs[0].MinSize()
+	min1 := objs[1].MinSize()
+
+	minWidth := fyne.Max(min0.Width, min1.Width)
+	return fyne.NewSize(minWidth, min0.Height+min1.Height).AddWidthHeight(0, theme.InnerPadding())
```

Comment:
No need for the extra add here. You can move it into the size creation.
```suggestion
	return fyne.NewSize(minWidth, min0.Height+min1.Height+theme.InnerPadding())
```